en/references/voip_security.md

```diff
@@ -0,0 +1,4 @@
+* *Guide:* ["VoIP Security Checklist"](https://www.computerworld.com/article/2567181/networking/the-voip-security-checklist.html) (ComputerWorld)
+* *Overview:* ["The Vulnerability of VoIP"](https://us.norton.com/voip-security-a-primer/article) (Symantec)
+* *Research:* ["Researchers find VoIP phones vulnerable to Simple Cyber attacks"](https://securityintelligence.com/news/researchers-find-voip-phones-vulnerable-to-simple-cyberattacks) (Security Intelligence)
+* *Tool:* ["Vsaudit (Eurialo)"](https://github.com/eurialo/vsaudit) (Eurialo)
```

Comment:
Reviewed this, it's great! can we add this: https://www.symantec.com/connect/articles/two-attacks-against-voip
which covers (Registration Hijacking) and (Eavesdropping) in details